lib/actions/ui.js

```diff
@@ -70,7 +70,8 @@ export function decreaseFontSize() {
       effect() {
         const state = getState();
         const old = state.ui.fontSizeOverride || state.ui.fontSize;
-        const value = old - 1;
+        // when the font-size is really small, below 5px, xterm starts showing up issues.
+        const value = old > 5 ? old - 1 : old;
```

Comment:
```suggestion
        const value = Math.max(old - 1, 5);
```
I prefer to use `max()` to floor a value, but this is a personal flavor.
Your way is totally acceptable.